### PR TITLE
Update persistance.lua

### PR DESCRIPTION
**Pull request by eltociear**
**Describe what the proposed change does**
- persistance -> persistence

### CODE REVIEW

File: CorsixTH/Lua/persistance.lua [hunk 1]
```diff
@@ -63,7 +63,7 @@ function unpermanent(name)
   saved_permanents[name] = nil
 end
 
-local --[[persistable:persistance_global_fetch]] function global_fetch(...)
+local --[[persistable:persistence_global_fetch]] function global_fetch(...)
```
Comment (lewri) [inline CorsixTH/Lua/persistance.lua:66]: I agree that the spelling is indeed unfortunate (even the file is spelt wrong). But there is a greater impact for amending a persistable function's name. I would hazard we don't change it unless we really have to.

File: CorsixTH/Lua/persistance.lua [hunk 2]
```diff
@@ -313,7 +313,7 @@ function LoadGame(data)
   TheApp.ui:onChangeResolution()
   -- Check if the blueish tone should be applied.
   -- Note: Blue filter control should be handled from world or ui, however when
-  -- loading a game we should let persistance do it.
+  -- loading a game we should let persistence do it.
```
Comment (lewri) [inline CorsixTH/Lua/persistance.lua:316]: Let's keep this though. No matter how small a change something ends up it's always appreciated.

**Review by lewri — CHANGES_REQUESTED** [review]
Appveyor did succeed it just made a fuss. Please revert the persistable rename.

File: CorsixTH/Lua/persistance.lua (hunk 1 above)
Comment (Alberth289346) [reply CorsixTH/Lua/persistance.lua:66]: Indeed, that name is used in old savegames, they will break if you change the name.

**Comment by tobylane** [comment]
You could change more of these. All but the first and last from https://grep.app/search?q=persistance&filter[repo][0]=CorsixTH/CorsixTH are worth trying out.

**Comment by lewri** [comment]
> You could change more of these. All but the first and last from [https://grep.app/search?q=persistance&filter[repo][0]=CorsixTH/CorsixTH](https://grep.app/search?q=persistance&filter%5Brepo%5D%5B0%5D=CorsixTH/CorsixTH) are worth trying out.

` corsixth.require("persistance")` also out with the `linux.yml` file, unless you wanted to change the filename.

File: CorsixTH/Lua/persistance.lua (hunk 1 above)
Comment (eltociear) [reply CorsixTH/Lua/persistance.lua]: Thanks! Reverted this.

**Review by lewri — APPROVED** [review]

**Pull request merged**